servicecontrol/installation-powershell.md

```diff
@@ -35,44 +52,98 @@ Get-Help Get-ServiceControlInstances
+
+Add-ServiceControlRemote -Name Test.ServiceControl -RemoteInstanceAddress http://localhost:44444/api
+```
 
-The following commands show how to remove a ServiceControl instance(s). To list existing instances of the ServiceControl service use `Get-ServiceControlInstances`.
+NOTE: The ServiceControl Audit instance must be configured with the transport address of a ServiceControl instance.
 
-Remove the instance that was created in the Add sample and delete the database and logs:
+NOTE: <Something about adding remotes>
+
+
+### Removing an instance
+
+Use the `Remove-ServiceControlInstance` cmdlet to remove a ServiceControl instance. 
 
 ```ps
 Remove-ServiceControlInstance -Name Test.ServiceControl -RemoveDB -RemoveLogs
 ```
 
-Remove all ServiceControl instances created in the Add sample and delete the database and logs for each one:
+In version 4 and above, use the `Remove-AuditInstance` cmdlet to remove a ServiceControl Audit instance.
 
 ```ps
-Get-ServiceControlInstances | Remove-ServiceControlInstance -RemoveDB -RemoveLogs
+Remove-ServiceControlRemote -Name Test.ServiceControl -RemoteInstanceAddress http://localhost:44444/api
+
+Remove-AuditInstance -Name Test.ServiceControl.Audit -RemoveDB -RemoveLogs
 ```
 
-There are additional parameters available to set additional configuration options such as forwarding queues, the transport connection string, or host name.
+NOTE: All connected ServiceControl Audit instances should be removed before removing the main ServiceControl instance. Use the `Get-ServiceControlRemotes` cmdlet to find a list of connected ServiceControl Audit instances for a given ServiceControl instance.
 
 
 ### Upgrading an instance
 
-The following command will list the ServiceControl instances current installed and their version number.
+The cmdlets in this section are used to upgrade the binaries of an existing instance. If the instance is running when the upgrade starts, it will be shut down during the upgrade and restarted once the upgrade is complete.
+
+Before the upgrade begins the configuration file of the existing version is examined to determine if all of the required settings are present. If a configuration setting is missing then the cmdlet will throw an error indicating the required additional parameter for the cmdlet.
+
+Use the `Invoke-ServiceControlInstanceUpgrade` cmdlet to upgrade a ServiceControl instance to the latest binaries.
+
+```ps
+Invoke-ServiceControlInstanceUpgrade -Name <Instance To upgrade>
+```
+
+Use the following to find a list of all of the ServiceControl instances and their version numbers:
 
 ```ps
 Get-ServiceControlInstances | Select Name, Version
 ```
 
-To upgrade an instance to the latest version of the binaries:
+Additional parameters may be required when upgrading an instance to version 4, to enable the creation of a new ServiceControl Audit instance.
```

Comment:
This is a bit difficult to review. Maybe just leave out the part after the comma?